CHANGELOG.md

```diff
@@ -9,7 +13,7 @@
 - Change self-enum color components with Dart builtin enums
 - Remove unused enum items (RGB, RGBA, HEX)
 - Move color components from `package:fff/parser.dart` to `package:fff/color.dart`
-- Restored tests to work with the new Dart library
+- Restored tests to work with the  Dart library
```

Comment:
Double space before `Dart library`. I think one of them is useless

Reply:
I put the new word back. victim of 'replace all' 'new' keyword.